programmes/management/commands/load_test_fixtures.py

```diff
@@ -92,31 +92,33 @@ def handle(self, *args, **options):
             siret=bailleur["siret"], defaults=bailleur
         )
 
-        if settings.ENVIRONMENT != "production":
+        if settings.ENVIRONMENT == "production":
+            raise Exception("il est interdit d'executer cette command en production")
+
             # Remove conventions and operation and lot
+        truncate_programme = False
+        inp = input("Do you want to truncate Conventions/Operations/Lots (N/y) ?")
+        if inp.lower() in ["y", "yes", "oui"]:
+            truncate_programme = True
+        elif inp.lower() in ["n", "no", "non"]:
             truncate_programme = False
-            inp = input("Do you want to truncate Conventions/Operations/Lots (N/y) ?")
-            if inp.lower() in ["y", "yes", "oui"]:
-                truncate_programme = True
-            elif inp.lower() in ["n", "no", "non"]:
-                truncate_programme = False
-            else:
-                print("Using default option: Operation won't be truncate")
-            if truncate_programme:
-                Programme.objects.all().delete()
-                Convention.objects.all().delete()
-
-            # Remove user
+        else:
+            print("Using default option: Operation won't be truncate")
```

Comment:
📖 Moment Bernard Pivot (enfin plus Shakespeare ici)

![Bernard](https://i.makeagif.com/media/5-11-2017/jQyRt4.gif)

Soit on dit "Operation won't be truncate**d**" soit, et je pense que c'est ce que tu voulais dire "Operation won't truncate (any table)"